### PR TITLE
YLP-241 Review openapi generation so we can serve it through a website

### PR DESCRIPTION
**Pull request by BigFab**
The deployed openapi documentation will be available in website hosted by S3

### CODE REVIEW

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Seems to work well.
Will see for the travis part on merge/publish.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Gatekeeper
 Authorization client and server for tidepool
 
+## 0.4.3 - 2020-10-26
```
Comment (yvon-dblg) [inline CHANGELOG.md:4]: Missing bump in package.json & package-lock.json

Comment (BigFab) [reply CHANGELOG.md:4]: reviewed

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**